tg-portfolio/components/contact/ContactForm.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+
+const ContactForm = () => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [subject, setSubject] = useState('');
+  const [message, setMessage] = useState('');
```

Comment:
Do it using a single state for a more clean structure:

```javascript
const [form, setForm] = useState({
   name: '',
   email: '',
   subject: '',
   message: '',
})
```

How to set the particular input field state value?

```javascript
value={form.email}
onChange={(e) => seform({...form, email: e.target.value})}
```

Reply:
<img width="905" alt="Screen Shot 2022-10-01 at 12 29 56 PM" src="https://user-images.githubusercontent.com/76760849/193423241-f7ce4e46-4a1f-4a54-92d0-67baccb03e2b.png">

I'm having trouble solving the JSON.stringify so it recognizes my data now that it has been changed to one single state. I've tried things like "form.name" and that also is not working. @TidbitsJS

Reply:
<img width="905" alt="Screen Shot 2022-10-01 at 12 57 02 PM" src="https://user-images.githubusercontent.com/76760849/193424206-96f1bb33-86c0-4fc7-9d5d-68a1b269253c.png">

Is this correct?

Reply:
Yes, that's what I wanted to see @t-creates :)